packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -294,6 +294,16 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		);
 	}
 
+	async findAllPotentiallyRunningExecutionIds() {
```

Comment:
1. Given the name, this should only return IDs?
2. We're not using the status so no need to select it?

Reply:
removed